pkg/inspector/image-inspector_test.go

```diff
@@ -60,12 +61,22 @@ func TestScanImage(t *testing.T) {
 				t.Errorf("%s should have succeeded but failed with %v", k, err)
 			} else {
 				resultFileContent, err := ioutil.ReadFile(v.s.ResultsFileName())
+				if err != nil {
+					t.Errorf("%s should have been able to read the"+
+						"results file but failed with: %v", k, err)
+				}
 				if string(resultFileContent) != string(report) {
-					t.Errorf("%s returned wrong results", k, err)
+					t.Errorf("%s returned wrong results: %v", k, err)
```

Comment:
thanks for updating the error check.  This still prints the error if the strings don't match but the error may be nil.  We should print the strings or if that is too big the message should read something like `html report on disk did not match the results from scanImage`.  Other than that I think this is good to go